### ISSUE TITLE
AAF fit_intercept=False on regression_dataset?

### ISSUE THREAD

**Issue by JonathanJHull** [CamDavidsonPilon/lifelines#332]
Hi,
In the quickstart section of the doc the following code snippet appears:
     aaf = AalenAdditiveFitter(fit_intercept=False)
     aaf.fit(regression_dataset, 'T', event_col='E')

Should fit_intercept by True since the regression dataset does not have column of ones?

The result is that when plotting the survival function with:  aaf.predict_survival_function(X.iloc[10:13]).plot() 
subject 10 has a 38% probability of surviving past 22 weeks.  

Is a print_summary() method on the way for AAF?

BTW, thanks very much for the excellent package.   It's an awesome addition to the python data science ecosystem.

**Comment by CamDavidsonPilon** [comment]
> Should fit_intercept by True since the regression dataset does not have column of ones?


You're right - this probably should have that flag set to True. I'll keep this issue open so I remember to update the docs.

> Is a print_summary() method on the way for AAF?

This has been asked quite a bit, and it's about time I do something about it (currently it doesn't exist). Can I ask what you would want to see in a `print_summary`?

> BTW, thanks very much for the excellent package. It's an awesome addition to the python data science ecosystem.

❤️ thanks!
